pkgs/development/python-modules/pytest-cram/default.nix

```diff
@@ -2,15 +2,16 @@
 
 buildPythonPackage rec {
   name = "${pname}-${version}";
```

Comment:
Please get rid of the `name` attribute. It isn't needed anymore for `buildPython*`.

Reply:
Ok, thanks. Done.